azerothcore/azerothcore-wotlk
# fix(DB/Factionchange): Added quests for T0.5 chain

With the help of: https://classic.wowhead.com/guides/dungeon-sets-1-2-quests-wow-classic

These quests are already in the database (so they aren't present in the SQL).
#(8960, 9032), #Bodley's Unfortunate Fate
#(8997, 8998), #Back to the Beginning



## Changes Proposed:
-  Added the missing alternative quests when doing a faction change for the T0.5 quest chain.

## Issues Addressed:

- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/10579

## SOURCE:


## Tests Performed:

- On a mage, did `.cha changefac` while completing one faction specific quest at a time with the help of `.q c [quest link]` and some `.damage 30000` macro.

## How to Test the Changes:


1. Choose a faction/class
2. Complete a faction specific quest (to quickly complete a quest, use the following command: `.q c [quest link]`)
3. Change faction (use the macro `.change changefaction Meow`)
4. See if you can grab the next quest from your faction equivalent NPC

Some useful macros
`.change changefaction Meow`
`.tele grommash`
`.tele greatforge`
`.tele gadgetza`
`.tele strath`
`.tele brs`

## Known Issues and TODO List:




## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
